### PR TITLE
add details to initial version

### PR DESCRIPTION
**Pull request by rrennick**
This PR adds more contextual information to two sections of the README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ Refer to https://github.com/woocommerce/woocommerce/blob/master/tests/e2e/docker
 
 ### Adding new tests
 
-To add new E2E tests, add new files to **`tests/e2e/specs/`** with the extension **`.test.js`**. The **`example.test.js`** may be used as a starting point.
+To add new E2E tests, add new files to **`tests/e2e/specs/`** with the extension **`.test.js`** or **`.spec.js`**. The **`example.test.js`** may be used as a starting point. Tests can be organized in folders `tests/e2e/specs/area-name/*.test.js`.
```
Comment (achyuthajoy) [inline README.md]: Looks good. How about changing:

Tests can be organized in folders **`tests/e2e/specs/area-name/*.test.js`** 

to

 Tests can also be organized in folders under the **`specs`** directory. For example: **`tests/e2e/specs/{folder-name}/example.test.js`**.

**Review by achyuthajoy — CHANGES_REQUESTED** [review]
Changes look good to me @rrennick. I've suggested a minor change to the wording. Let me know your thoughts.

File: README.md (hunk 1 above)
Comment (rrennick) [reply README.md]: @achyuthajoy Thanks for the suggestion. I've updated the text.

**Review by achyuthajoy — APPROVED** [review]
Looks good to me @rrennick 👍 

Merging

**Pull request merged**